pyrgg/functions.py

```diff
@@ -28,6 +28,53 @@ def is_weighted(max_weight, min_weight, signed):
     return True
 
 
+def get_min_max_weight(weight_dic):
+    """
+    Get minimum and maximum weight values.
```

Comment:
This function's docstring is incorrect.

Reply:
Oh shoot. Yes, that's again residual from the last commits. fixed in [cc0f41f](https://github.com/sepandhaghighi/pyrgg/pull/173/commits/cc0f41f339eb1cfd0dfdcd16007928967c2a8c88).